book/src/rent.md

```diff
@@ -0,0 +1,61 @@
+# Rent
+
+Accounts on Solana may have owner-controlled state (`Account::data`) that's separate 
+from the account's balance (`Account::lamports`).  Since validators on the network need 
+to maintain a working copy of this state in memory, the network charges a time-and-space 
+based fee for this resource consumption, also known as Rent.
+
+## Two-tiered rent regime
+
+Accounts which maintain a minimum balance equivalent to 2 years of rent are exempt from
+rent.  Accounts whose balance falls below this threshold are charged rent 
+at a rate specified in genesis, in lamports per kilobyte per year.
+
+## Collecting rent
+
+A field on each account (`Account::rent_due_slot`) keeps track of the next slot at which the account will 
+owe rent, similar to a time-to-live.
+
+Rent is charged directly from the account balance via issuance of `SystemInstruction::RentDue` and is deposited into 
```

Comment:
Does this appear in the block?  There will be a way to many of these.

Reply:
yes, that's the proposal.  one tx per account per epoch for accounts below minimum balance

Reply:
periodic scans of accounts_db could be required instead, but that seemed expensive

another option is charging rent when the account is accessed, but credit-only accounts get in the way of that, and there's no way to clean up old, unused accounts

Reply:
If you think we can fit the transactions into the ledger then a scan will be fast :)

Reply:
scan at epoch boundary also has the not-nice property of that first leader getting all the rent, I could destroy the rent instead.  thoughts?

Reply:
@rob-solana we can credit everyone in the epoch

Reply:
stake-weighted distribution?

Reply:
Yep.  @ericlwilliams any reason to burn half of it?

Reply:
The way I think of it is that burning fees/rent is a network wide 'reward', while distribution to active validators is, obvious, a local reward. In this case, I think of state storage as a network wide externality, in which case I think burning some portion probably makes sense.

Reply:
will you update the economics section of the book with this and add rationale?  I can't think of a burn incentive, collecting rent won't be optional, not subject to censorship.

Reply:
I'm not thinking of it as an incentive, but more about how you distribute the externality across the network. If I'm thinking about this the right way, without burning, the payment of the externality would go just to the validators. Burning some portion is somewhat like re-distributing those payments across to the non-validator network participants as well, who also bear the costs of long-term storage.
I'll update the economics section with rational from whatever we land on out of this discussion :)

Reply:
ok, I'm convinced.  50% by default?

Reply:
sgtm